core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/UserImpl.java

```diff
@@ -54,7 +57,7 @@
  * @since 0.0.2
  */
 @Entity
-@Table(name = "User")
+@Table(name = "`User`")
```

Comment:
Just out of curiosity, were the backticks required because of 'USER' being a keyword?